src/plugins/data/common/search/aggs/metrics/count_fn.ts

```diff
@@ -78,10 +77,7 @@ export const aggCount = (): FunctionDefinition => ({
         enabled,
         schema,
         type: METRIC_TYPES.COUNT,
-        params: {
-          ...rest,
-          json: getParsedValue(args, 'json'),
-        },
+        params: rest,
```

Comment:
We should remove the `json` argument from the function definition above as well, since it's no longer being used.

Note that this might cause a TS error because the function definition is using `AggExpressionFunctionArgs` which is pulling the relevant type from the `AggParamsMapping`. For `count`, the value used is `BaseAggParams` which includes json: https://github.com/elastic/kibana/blob/bf0f6fa574f4d998510f4fb38ea9aaab0d4f1db4/src/plugins/data/common/search/aggs/types.ts#L138

If TS gives you trouble, you may need to define an `AggParamsMetricCount` in the `count.ts` file without extending `BaseAggParams` as others agg types do.

Reply:
Thanx @lukeelmers , I missed that, TS looks happy btw :)